installation-bundle/src/Database/Installer.php

```diff
@@ -47,13 +52,37 @@ public function __construct(Connection $connection, DcaSchemaProvider $schemaPro
     /**
      * @return array<string>
      */
-    public function getCommands(): array
+    public function getCommands(bool $byGroup = true): array
```

Comment:
I don't think we use typed argument in the legacy code. Not sure if @leofeyer still would use `$blnUseGroups` names etc.

Reply:
Legacy code?

Reply:
Since the class is not in `src/Resources/contao`, it is not a legacy class. The code is fine IMHO.